### PR TITLE
update of jupyter nbs

### PR DESCRIPTION
**Pull request by AliceBalfanz**
Updated juypter notebooks so they can run automatically with appvayor CI:
Changes made: Subsetting the the geopandas dataframe which is written into the collection, and deleting the collection at the end of each notebook so each nb can run independently.

### CODE REVIEW

File: notebooks/geodb_share_collections.ipynb [hunk 1]
```diff
@@ -262,28 +336,149 @@
...
      },
-     "execution_count": 6,
+     "execution_count": 12,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "geodb = GeoDBClient(client_id=\"93wdyTwOJebDozSCB4CubsWdFVKrSaXY\", client_secret=\"zv0h3D3s90KAL640bw_MU4XMb6DfpJQloh877mH6Z1Nu4NrgPU2wSG3Ur_poUtqy\")\n",
+    "geodb = GeoDBClient(client_id=\"93wdyTwOJebDozSCB4CubsWdFVKrSaXY\", client_secret=\"zv0h3D3s90KAL640bw_MU4XMb6DfpJQloh877mH6Z1Nu4NrgPU2wSG3Ur_poUtqy\", auth_mode=\"client-credentials\")\n",
```
Comment (dzelge) [inline notebooks/geodb_share_collections.ipynb:345]: Maybe we should change this. I know I did that  to test whether the sharing works. You could try and set them also via env variables. Something like this:

```python
test_client_id = os.environ.get("TEST_CLIENT_ID")
test_client_secret = os.environ.get("TEST_CLIENT_SECRET")

geodb = GeoDBClient(client_id=test_client_id, client_secret=test_client_secret, auth_mode="client-credentials")

```

File: notebooks/geodb_share_collections.ipynb [hunk 2]
```diff
@@ -459,79 +655,55 @@
...
-     "execution_count": 43,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "geodb = GeoDBClient(client_id=\"93wdyTwOJebDozSCB4CubsWdFVKrSaXY\", client_secret=\"zv0h3D3s90KAL640bw_MU4XMb6DfpJQloh877mH6Z1Nu4NrgPU2wSG3Ur_poUtqy\")\n",
-    "geodb.whoami"
+    "#geodb = GeoDBClient(client_id=\"93wdyTwOJebDozSCB4CubsWdFVKrSaXY\", client_secret=\"zv0h3D3s90KAL640bw_MU4XMb6DfpJQloh877mH6Z1Nu4NrgPU2wSG3Ur_poUtqy\", auth_mode=\"client-credentials\")\n",
```
Comment (dzelge) [inline notebooks/geodb_share_collections.ipynb:669]: See above

**Review by dzelge — CHANGES_REQUESTED** [review]
Small changes, see below re test credentials we used sonewhere in one of teh notebooks.

**Comment by AliceBalfanz** [comment]
@dzelge : I changed the usage of the test credentials according to your suggestion :)

**Pull request merged**